src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -2549,19 +2589,103 @@ public Response resetFileStore(@PathParam("identifier") String dvIdtf,
         }
         if (!user.isSuperuser()) {
             return error(Response.Status.FORBIDDEN, "Superusers only.");
-    	}
-        
-        Dataset dataset; 
-        
+        }
+
+        Dataset dataset;
+
         try {
             dataset = findDatasetOrDie(dvIdtf);
         } catch (WrappedResponse ex) {
             return error(Response.Status.NOT_FOUND, "No such dataset");
         }
-        
+
         dataset.setStorageDriverId(null);
         datasetService.merge(dataset);
-    	return ok("Storage reset to default: " + DataAccess.DEFAULT_STORAGE_DRIVER_IDENTIFIER);
+        return ok("Storage reset to default: " + DataAccess.DEFAULT_STORAGE_DRIVER_IDENTIFIER);
+    }
+
+    @GET
+    @Path("{identifier}/curationLabelSet")
+    public Response getCurationLabelSet(@PathParam("identifier") String dvIdtf,
+            @Context UriInfo uriInfo, @Context HttpHeaders headers) throws WrappedResponse {
+
+        Dataset dataset;
+
+        try {
+            dataset = findDatasetOrDie(dvIdtf);
+        } catch (WrappedResponse ex) {
+            return error(Response.Status.NOT_FOUND, "No such dataset");
+        }
+
+        return response(req -> ok(dataset.getEffectiveCurationLabelSetName()));
```

Comment:
I'm not sure how I feel about the lack of authentication on this GET method. We want it to work on drafts, for example?

Reply:
FWIW: This is nominally just the name of the set that's allowed, not the particular state/curation label on this dataset, but I added auth - not sure there's a reason anyone but superuser needs to know this.

Reply:
Perhaps a curator would like to know but I think superuser is better than no auth at all. Thanks.